calculation/gmhazard_calc/gmhazard_calc/dbs/IMDB.py

```diff
@@ -3,6 +3,7 @@
 from contextlib import contextmanager
 from typing import Optional, Dict, List, Union
 
+import gmhazard_calc.im
```

Comment:
I think this needs to be with other internal modules +
a bit ambiguous with line number 14.